src/utils/batt_utils.ts

```
@@ -0,0 +1,231 @@
+    // Try a direct approach with osascript first - this works reliably based on our testing
+    try {
+      console.log("Using osascript to run batt command...");
+      // Create a temporary file for the output to avoid issues with Terminal output parsing
+      const tempOutputFile = `/tmp/batt_status_${Date.now()}.txt`;
+      const osascriptCmd = `/usr/bin/osascript -e 'do shell script "${battCmd} status > ${tempOutputFile} 2>&1"'`;
+      
+      // Execute the command via osascript
+      execSync(osascriptCmd, { encoding: 'utf8' });
+      
+      // Read the output from the temporary file
+      const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)